apps/api/infra/api.ts

```diff
@@ -244,6 +244,17 @@ export const serviceSetup = (services: {
           'clustercfg.general-redis-cluster-group.whakos.euw1.cache.amazonaws.com:6379',
         ]),
       },
+      LICENSE_SERVICE_REDIS_NODES: {
+        dev: json([
+          'clustercfg.general-redis-cluster-group.5fzau3.euw1.cache.amazonaws.com:6379',
+        ]),
+        staging: json([
+          'clustercfg.general-redis-cluster-group.ab9ckb.euw1.cache.amazonaws.com:6379',
+        ]),
+        prod: json([
+          'clustercfg.general-redis-cluster-group.whakos.euw1.cache.amazonaws.com:6379',
+        ]),
+      },
```

Comment:
You have a few options about setting redis envs.

1. You could get rid of these envs (and all other redis envs already defined on the service) by just chaining `.redis()`  on the service-builder object. That will[ automatically generate](https://github.com/island-is/island.is/blob/main/infra/src/dsl/redis.spec.ts#L36) `REDIS_URL_NODE_01=redis-applications.internal:6379` env in your containers. Would require updating process.env references in your application.

2. use the `redis-applications.internal:6379` directly. For IDS it's `redis-ids.internal:6379`. These envs are internal dns names accessible in all environments and are mapped to corresponding endpoints.